CMakeLists.txt

```diff
@@ -20,6 +20,15 @@ set_property(TARGET boost PROPERTY
 set_property(TARGET boost PROPERTY
     INTERFACE_LINK_LIBRARIES ${Boost_LIBRARIES})
 
+# Include Folly as an imported target
+find_package(Folly 2017.07.31.00 REQUIRED)
```

Comment:
Can we pin to a less specific version of Folly? For example, the current version of Folly in homebrew is already a `2017.08.14.00`.